tech/languages/rust/further-reading.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Further reading
+subsection: rust
+order: 4
+---
+
+# Learning resources
```

Comment:
I wouldn't start with this.
It can be at the bottom of some page as `references` (I'm not sure about the book though).
This 'portal' aims to have `quickstarts` and `references` useful for *Developers on Fedora* (Fedora-specifics being the main part). Please give a quick check to these:

https://fedoraproject.org/wiki/Websites/Developer#About_The_Project
http://developer-portal.github.io/contributing/write-introduction.html

Reply:
This is a whole section with references. I think it is much more useful to include links to the upstream project than to write something new. This helps by pointing developers in the right direction.

Reply:
Yes, I can see that. I'd would prefer something like this, instead:

 - "main page" (basic Fedora install + basic info + general references)
 - "arbitrary-topic-page" (could be "cargo" or "ides", basic install & info, topic-related references  -- one topic per page!)
 - "sig-or other page" (contacts)

Reply:
Note: we also prefer to link the content instead of duplicating it. It's not about the content but a structure of it.

Reply:
IOW everyone will skip this page if it's not short and per-purpose structured.

Reply:
This could be an "About page". With some short intro, links to the other sections (incl. the "install rust" one), and references to books etc. at the bottom.